### PR TITLE
Drop support for Node 13; Drop support for Ember < 3.12; Update ember-cli blueprint to latest;

### PR DESCRIPTION
**Pull request by loganrosen**
I've used ember-cli-update to update this addon to the ember-cli 3.25
blueprint and run all of the latest codemods.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,44 +19,67 @@
...
-    "test:all": "ember try:each"
+    "lint:js:fix": "eslint . --cache --fix",
+    "start": "ember serve",
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
     "ember-cli-babel": "^7.23.1",
+    "ember-cli-htmlbars": "^5.3.2",
```
Comment (rwjblue) [inline package.json:36]: Can you double check to see if we have any `addon/**/*.hbs` files? If not, can you move this back to devDeps?

**Review by rwjblue — COMMENTED** [review]
Looks good overall, thanks for working on it!

File: package.json (hunk 1 above)
Comment (loganrosen) [reply package.json]: Fixed. I wasn't sure if the fact that it provides a helper meant that it should have an `ember-cli-htmlbars` dependency.

Comment (rwjblue) [reply package.json]: Nope, you only need it as a dependency if you have handlebars files.

**Review by rwjblue — APPROVED** [review]
Thank you!

**Pull request merged**